### PR TITLE
fix(twig): normalize twig templates, pt3 - INNO-1019

### PR DESCRIPTION
**Pull request by yhuard**
Updated:
- component-form-label
- component-form-legend
- component-form-radio
- component-form-radio-group

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 13e46a0b8193671c7b6556d15eaae87525d90619

https://deploy-preview-649--europa-component-library.netlify.com

File: src/generic/generic-component-form-radio-group/generic-component-form-radio-group.twig [hunk 1]
```diff
@@ -1,28 +1,48 @@
...
+    - "system" (string) (default: "generic")
 #}
 
-{% set id = name|default('ecl-radio-1') %}
-{% set name = name|default('ecl-radio') %}
-{% set desc = desc|default('Description') %}
-
 {# Internal properties #}
 
+{% set _id = name|default('ecl-radio-1') %}
```
Comment (kalinchernev) [inline src/generic/generic-component-form-radio-group/generic-component-form-radio-group.twig]: `{% set _id = id|default('ecl-radio-1') %}` I think

File: src/systems/eu/eu-component/eu-component-form/eu-component-form-label/eu-component-form-label.twig [hunk 2]
```diff
@@ -1,21 +1,38 @@
...
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
   {% for attr in extra_attributes %}
     {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
   {% endfor %}
 {% endif %}
 
-<label class="{{ _css_class }}"{{ _extra_attributes|raw }}{{ for_attribute is defined ? ' for="' ~ for_attribute ~ '"'  : '' }}>{% block label label %}</label>
+{# Print the result #}
+
+<label class="{{ _css_class }}"{{ _extra_attributes|raw }}{{ _for_attribute is not empty ? ' for="' ~ _for_attribute ~ '"'  : '' }}>{% block label _label %}</label>
```
Comment (kalinchernev) [inline src/systems/eu/eu-component/eu-component-form/eu-component-form-label/eu-component-form-label.twig]: To me it seems eu-component-form/eu-component-form-label/eu-component-form-label.twig is same as /src/generic/generic-component-form-label/generic-component-form-label.twig should it be `{% include '@ecl/generic-component-form-label' with { system: 'eu' } %}`?

File: src/systems/eu/eu-component/eu-component-form/eu-component-form-legend/eu-component-form-legend.twig [hunk 3]
```diff
@@ -1,27 +1,46 @@
+{#
```
Comment (kalinchernev) [inline src/systems/eu/eu-component/eu-component-form/eu-component-form-legend/eu-component-form-legend.twig]: This twig template also looks to be exact as the generic one.

**Review by kalinchernev — DISMISSED** [review]
Maybe I have missed a detail in the topic of system-specific twig templates with same structure as the generic one, I was basing my comments on https://next--europa-component-library.netlify.com/ec/docs/conventions/component#systems-component

**Comment by kalinchernev** [comment]
There's some additional feedback from the CI:
```
[chrome 60.0 Windows 10 #0-7] Session ID: 3c38669280374f8abd797ee3b0257b4c
[chrome 60.0 Windows 10 #0-7] Spec: /drone/src/github.com/ec-europa/europa-component-library/src/systems/ec/ec-template/ec-template-filter/test/spec/filters.js
[chrome 60.0 Windows 10 #0-7] Running: chrome (v60.0) on Windows 10
[chrome 60.0 Windows 10 #0-7]
[chrome 60.0 Windows 10 #0-7] filters
[chrome 60.0 Windows 10 #0-7]   ✓ should match the reference screenshot
[chrome 60.0 Windows 10 #0-7]   1) should be accessible
[chrome 60.0 Windows 10 #0-7]
[chrome 60.0 Windows 10 #0-7]
[chrome 60.0 Windows 10 #0-7] 1 passing (3m, 27s)
[chrome 60.0 Windows 10 #0-7] 1 failing
[chrome 60.0 Windows 10 #0-7]
[chrome 60.0 Windows 10 #0-7] 1) filters should be accessible:
[chrome 60.0 Windows 10 #0-7] expected a11y report to be empty but instead got: - [moderate] Ensures every id attribute value is unique: -- [moderate] .ecl-form: --- [moderate] Document has multiple elements with the same id attribute: form-filter
[chrome 60.0 Windows 10 #0-7] AssertionError: expected a11y report to be empty but instead got: - [moderate] Ensures every id attribute value is unique: -- [moderate] .ecl-form: --- [moderate] Document has multiple elements with the same id attribute: form-filter
[chrome 60.0 Windows 10 #0-7]     at Context.it (/drone/src/github.com/ec-europa/europa-component-library/src/systems/ec/ec-template/ec-template-filter/test/spec/filters.js:25:29)
[chrome 60.0 Windows 10 #0-7]     at new Promise (<anonymous>)
[chrome 60.0 Windows 10 #0-7]     at new F (/drone/src/github.com/ec-europa/europa-component-library/node_modules/core-js/library/modules/_export.js:36:28)
[chrome 60.0 Windows 10 #0-7]
[chrome 60.0 Windows 10 #0-7] Check out job at https://saucelabs.com/tests/3c38669280374f8abd797ee3b0257b4c
[chrome 60.0 Windows 10 #0-7]
```

File: src/generic/generic-component-form-label/generic-component-form-label.twig [hunk 4]
```diff
@@ -1,21 +1,38 @@
 {#
   Parameters:
-    - "label" (string or block): label (mandatory)
-    - "for_attribute" (string): ID of the element (optional)
-    - "extra_classes" (string): extra CSS classes to be added
-    - "extra_attributes" (array): extra attributes classes (optional, format: [{ 'name': 'name_of_the_attribute', 'value': 'value_of_the_attribute'}])
+    - "label" (string) (default: '')
+    - "for_attribute" (string) (default: ''): ID of the element
```
Comment (degliwe) [inline src/generic/generic-component-form-label/generic-component-form-label.twig]: nitpicking: 
"ID of the element the label is bound to" seems more useful

File: src/generic/generic-component-form-label/generic-component-form-label.twig [hunk 5]
```diff
@@ -1,21 +1,38 @@
...
+    - "extra_classes" (string) (default: ''): additional classes to add to the component
```
Comment (degliwe) [inline src/generic/generic-component-form-label/generic-component-form-label.twig]: nitpicking:
"extra CSS classes to be added" doesn't really add to the name of the param

File: src/generic/generic-component-form-legend/generic-component-form-legend.twig [hunk 6]
```diff
@@ -1,27 +1,46 @@
+{#
+  Parameters:
+    - "content" (string) (default: '')
+    - "level" (string) (default: ''): none, "1" or "2"
```
Comment (degliwe) [inline src/generic/generic-component-form-legend/generic-component-form-legend.twig]: nitpicking:
"none, '1' or '2' " could be completed. is "none" a valid value?  
"1": Legend for a top-level fieldset 
"2": Legend for a nested fieldset

**Review by degliwe — COMMENTED** [review]
Let's review quickly my comments and decide what to do.

**Review by degliwe — APPROVED** [review]

**Pull request merged**